cmd/argo/commands/resubmit.go

```diff
@@ -40,7 +40,7 @@ func NewResubmitCommand() *cobra.Command {
 				defer conn.Close()
 				apiGRPCClient, ctx := GetWFApiServerGRPCClient(conn)
 				errors.CheckError(err)
-				wfReq := workflow.WorkflowGetRequest{
+				wfReq := workflowpkg.WorkflowGetRequest{
 					Namespace: namespace,
 					Name:      args[0],
 				}
```

Comment:
The rest of the implementation still remains the old way of doing.  Can it not be refactored?

Reply:
I'd like to do it piecemeal - as a lot of the CLI code has no automated test - so refactoring is more likely to introduce bugs and regressions. 

Whenever we need to change the CLI, we should complete the appropriate part of the client and migrate.